packages/blade/src/components/TopNav/TopNav.web.tsx

```diff
@@ -73,21 +40,19 @@ const TopNav = ({ children, ...styledProps }: TopNavProps): React.ReactElement =
     <TopNavContext.Provider value={{ backgroundColor: styledProps.backgroundColor }}>
       <Box
         display="grid"
-        gridTemplateColumns={{ base: 'minmax(0, 1fr) auto', m: 'auto minmax(0, 1fr) auto' }}
+        gridTemplateColumns="auto minmax(0, 1fr) auto"
         alignItems="center"
         position="sticky"
         top="0px"
         width="100%"
-        paddingX="spacing.3"
+        paddingY={{ base: 'spacing.3', m: 'spacing.0' }}
+        paddingX={{ base: 'spacing.4', m: 'spacing.3' }}
         height={makeSize(TOP_NAV_HEIGHT)}
         zIndex={1}
```

Comment:
We should take it from consumer right? and define a better default on our end

Reply:
🤔 already doing that only. this is the default padding which is set on figma. 
And letting consumers override also just in case.

Reply:
Not the padding. I meant zIndex

Reply:
done